bundles/override.rst

```diff
@@ -105,17 +105,16 @@ associations. Learn more about this feature and its limitations in
 Forms
 -----
 
-In order to override a form type, it has to be registered as a service (meaning
-it is tagged as ``form.type``). You can then override it as you would override any
-service as explained in `Services & Configuration`_. This, of course, will only
-work if the type is referred to by its alias rather than being instantiated,
-e.g.::
+Form types are referred to by their fully-qualified class name::
```

Comment:
But we may want to remove this paragraph entirely for the 2.7 docs and only talk about form type extensions.